plugins/operators/__init__.py

```diff
@@ -1416,6 +1416,202 @@ def execute(self, ctx):
         ctx.dataset.create_index(field_name, unique=unique, wait=False)
 
 
+class ConvertIndexToUnique(foo.Operator):
+    @property
+    def config(self):
+        return foo.OperatorConfig(
+            name="convert_index_to_unique",  # Must match what's in fiftyone.yml
+            label="Convert an existing index to unique",
+            description="Convert an existing non-unique index to a unique index in the dataset if possible or identify any unique key violations that would occur.",
+            allow_immediate_execution=True,
+            allow_delegated_execution=True,
+            default_choice_to_delegated=False,
+        )
+
+    def resolve_input(self, ctx):
+        import re
```

Comment:
nit: prefer builtin module imports to be added at the top-level. Local imports should be reserved for special cases where a top-level import isn't possible